src/Core/Transform/DeformAVolumeWithAThinPlateSpline/Documentation.rst

```diff
@@ -0,0 +1,59 @@
+:name: DeformAVolumeWithAThinPlateSpline
+
+Deform a Volume With a Thin Plate Spline
+================================
```

Comment:
Warning reported by the CI build:
https://open.cdash.org/viewBuildError.php?type=1&buildid=8380872

Reply:
:eagle: :eye:

Reply:
Wonderful. Thanks Matt :green_heart:. Resolving.